ansible_collections/arista/avd/roles/eos_designs/templates/network_services/ip-igmp-snooping.j2

```diff
@@ -13,10 +13,13 @@ ip_igmp_snooping:
 {# Detect if a svi_profile exists #}
 {# If exists, create a shortpath to access profile data #}
 {%                 if svi.profile is arista.avd.defined %}
-{%                     set svi_profile = svi_profiles[svi.profile] | arista.avd.default() %}
+{%                     set svi_profile = svi_profiles | arista.avd.default() |
```

Comment:
```suggestion
{%                     set svi_profile = svi_profiles | arista.avd.default([]) |
```

Reply:
@Shivani-chourasiya now you changed it from `({})` to `()`. This will return `None` so it will fail on `selectattr` if the `svi_profiles` var is not defined.

Reply:
Corrected!